JavaAssignment/src/rahulM/LargestNum.java

```diff
@@ -0,0 +1,33 @@
+//Find largest of three numbers using logical And operator
+package switchcase;
+
+import java.util.Scanner;
+
+public class LargestNum {
+
+	public static void main(String[] args) {
+		 LargestNum lrgNum = new LargestNum();
+		 lrgNum.largestNumber();
+		
+	}
+	void largestNumber(){
+		int x, y, z;
+	      System.out.println("Enter three integers ");
+	      Scanner sc = new Scanner(System.in);
+	 
+	      x = sc.nextInt();
+	      y = sc.nextInt();
+	      z = sc.nextInt();
+	 
+	      if ( x > y && x > z )
+	         System.out.println(x+"  is largest");
+	      else if ( y > x && y > z )
```

Comment:
Here, not need to compare again x & y.
if x > y and x > z then x is large
then no need to compare again x with y
second if condition.